include/OpenColorIO/OpenColorIO.h

```diff
@@ -161,6 +161,17 @@ extern OCIOEXPORT void ResetToDefaultLoggingFunction();
 /// Log a message using the library logging function.
 extern OCIOEXPORT void LogMessage(LoggingLevel level, const char * message);
 
+/**
+ * @brief Set the Compute Hash Function object to use; otherwise, use the
+ * default.
+ * 
+ * @param hashFunction 
```

Comment:
Please remove the two following lines. BTW the second line is in fact wrong. `OCIOEXPORT` is not the return type but a macro to export or not the function C++ symbol.

Reply:
Most of this brief was generated by my IDE. Completely missed out on this! I'll surely fix it sir.